sale_order_project/models/sale.py

```diff
@@ -59,8 +65,34 @@ def action_create_project(self):
         project_obj = self.env['project.project']
         for order in self:
             vals = self._prepare_project_vals(order)
-            project = project_obj.create(vals)
-            order.write({
-                'project_id': project.analytic_account_id.id
-            })
-        return True
+            if order.project_template_id:
+                new_date_start = time.strftime('%Y-%m-%d')
```

Comment:
It is more correct to use `fields.Date. context_today()`. See http://www.odoo.com/documentation/10.0/reference/orm.html